ReactAndroid/src/main/java/com/facebook/react/views/art/ARTShapeShadowNode.java

```diff
@@ -164,6 +170,52 @@ protected boolean setupStrokePaint(Paint paint, float opacity) {
     return true;
   }
 
+  /*
+   * calculate stops count from array
+   * from array minus first 5 elements because it 1 - LinearGradient key, 2,3,4,5 - LinearGradient position
+   * and after devide on 5 because 1,2,3,4 - colors A,R,G,B and 5 - stop per 4 colors.
+   */
+  private static int getStopsCount(float[] value) {
+    int count = (value.length - 5)/5;
+    return count;
+  }
+
+  /*
+   * sorting stops and stopsColors from array
+   */
+  private static int gradientStopsParsing(float[] value, int stopsCount, float[] stops, int[] stopsColors) {
```

Comment:
Method naming convention in java is to start with a verb. E.g., parseGradientStops

Besides it looks like you're not using the result of that method so I recommend to change the return type to void